tasks/redhat.yml

```diff
@@ -0,0 +1,20 @@
+- name: Add firewalld rules
```

Comment:
Could you please rename this file to `firewall.yml`. This can be conditionally included from the `main.yaml` file using an appropriately name variable (added to `default.yaml` and the README), for people wanting to modify firewall configurations.

Reply:
Please default to false for backwards compatibility purposes.